drivers/vmwarevsphere/vsphere.go

```diff
@@ -965,6 +970,51 @@ func (d *Driver) Remove() error {
 
 	f.SetDatacenter(dc)
 
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
+	if err != nil {
+		return err
+	}
+
+	// grab /home/docker/.ssh/authorized_keys and compare with id_rsa.pub
+	opman := guest.NewOperationsManager(c.Client, vm.Reference())
+
+	fileman, err := opman.FileManager(ctx)
+	if err != nil {
+		return err
+	}
+
+	src := d.ResolveStorePath("auth_keys")
+	auth := AuthFlag{}
+	auth.auth.Username = B2DUser
```

Comment:
should use defaultSSHUser and defaultSSHPass, my updates will make everything standardize to this.

Reply:
Updated, thanks